k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapConnectionTest.java

```diff
@@ -901,6 +901,39 @@ public void sendContinuation() throws Exception {
         server.verifyInteractionCompleted();
     }
 
+    @Test
+    public void executeSingleCommand_on_openConnection_shouldReturnResult() throws Exception {
```

Comment:
Why "openConnection"? I suggest the name `executeSingleCommand_withOkResponse_shouldReturnResponse`.